src/game/Hero.ts

```diff
@@ -0,0 +1,47 @@
+export class Hero {
+  // Не понимаю, что делать с типами, чтобы не было такой большой колонки
+  x: number;
+  y: number;
+  radius: number;
+  inBoost: boolean;
+  boost: number;
+  speed: number;
+  lose: boolean;
```

Comment:
❗️
Для булева значения лучше подойдёт
```suggestion
  lost: boolean; // isLost, isLoser, etc
```